client/packages/programs/src/JsonForms/components/Prescription/StockLineTable.tsx

```diff
@@ -34,32 +36,51 @@ export const StockLineTable = ({
 
   const columns = useColumns<DraftStockOutLine>(
     [
+      {
+        width: 60,
+        key: 'expiry',
+        label: 'label.expiry',
+        accessor: ({ rowData }) => rowData.expiryDate,
+        Cell: ExpiryDateCell,
+        align: ColumnAlign.Right,
+      },
```

Comment:
Using correct `ExpiryDateCell` component now, which displays expired lines in red.

Reply:
Also, now that the date is right-aligned, it looks better as the first column